code/modules/atmospherics/machinery/components/unary_devices/vent_pump.dm

```diff
@@ -125,8 +125,9 @@
 
 /obj/machinery/atmospherics/components/unary/vent_pump/Destroy()
 	var/area/A = get_area(src)
-	A.air_vent_names -= id_tag
-	A.air_vent_info -= id_tag
+	if (A)
+		A.air_vent_names -= id_tag
+		A.air_vent_info -= id_tag
```

Comment:
Where is get_area() failing? This shouldn't be happening.

Reply:
The RPD creates atmos components in nullspace in order to read their InitDirections